### ISSUE TITLE
unable to enter waypoints LL, PBD, PBX or PD type.

### ISSUE THREAD

**Issue by ccapilot** [flybywiresim/a32nx#4376]
**Is your feature request related to a problem? Please describe.**
As per the title, unable to enter waypoints using either LL (full lat/longs or abreviated lat/longs) or PBD (Place/bearing/distance), PBX (place-bearing/place-bearing) or PD (Place/distance or along track) type waypoints.

**Describe the feature you'd like to see implemented**
This is very important feature to have, so I hope to see this implemented, the ability to enter lat/longs in they different formats as well place/bearing/distance and place/bearing/place/bearing type waypoints.

**References**
FCOM search for waypoint identification

**Additional context**

Discord username (if different from GitHub):

**Comment by tracernz** [comment]
Partially duplicates #1380, but this issue is more complete. I have implemented the logic to parse these formats from the scratchpad in #3752, but the flight plan side of it probably won’t come before a new flightplan manager.

**Comment by MisterChocker** [comment]
Yeah those are pilot waypoints, before we can start work on this we need to implement our flight plan manager first.
